include/reloader.h

```diff
@@ -42,7 +42,7 @@ class Reloader {
 	/// method creates a temporary handle that is destroyed when method
 	/// completes.
 	void reload(unsigned int pos,
-		unsigned int max = 0,
+		bool show_progress = false,
```

Comment:
It looks like `show_progress` serves the same purpose as `unattended`. I guess that `max == 0` case wasn't actually important — it's inconceivable that a user would reload something if `FeedContainer` has no feeds in it. My suggestion is to drop `show_progress` altogether, and remove `max` from the doc comment.

Reply:
The `max == 0` case indeed could happen when `FeedContainer` has no feeds in it.
However, there are more opportunities for `max == 0` (or with the current changes, `show_progress == false`):
- https://github.com/newsboat/newsboat/blob/9370e77d476853bb738bdf3579843d9d1975da1d/src/feedlistformaction.cpp#L121
- https://github.com/newsboat/newsboat/blob/9370e77d476853bb738bdf3579843d9d1975da1d/src/itemlistformaction.cpp#L412

I prefer to keep the current changes, but I'm fine with changing it if you want to always show `[<pos>/<num_feeds>]` in the status (currently it is not included when the user executes a single `reload` operation).

I did update the doc comment so it now mentions `show_progress` instead of `max`.

Reply:
Oh, sorry, I misunderstood the logic ._. `show_progress` and `unattended` are indeed different: the former is only responsible for the `<pos>/<num_feeds>` part, while the latter can show/hide the whole message.

The changes to the doc comment look good; no other changes needed. Sorry for wasting your time on this!

Reply:
No problem, thank you for the review!